app/utils/translation.py

```diff
@@ -1,13 +1,17 @@
 import json
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
And here I would also set the level to INFO instead of DEBUG. 

In the future, we can refactor the loggers so that we can choose the level of logging dynamically when we run the app, for example with and environmental variable. That way, we can easily get the logs that are useful for us at that moment (info, error, debug, etc.). I suggest INFO here as default because it is more standard.

Reply:
check!